tests/cases/dom/manipulate/toggleClass.js

```diff
@@ -33,6 +33,16 @@ QUnit.test(
 
         toggleClass(element, "foo", false);
         assert.strictEqual(element.classList.length, 0, "class wasn't removed with toggleClass");
+
+        toggleClass(element, ["foo", "bar"], true);
+        assert.strictEqual(element.classList.length, 2, "adding an array of classes should add all classes");
+        assert.ok(element.classList.contains("foo"));
+        assert.ok(element.classList.contains("bar"));
```

Comment:
Can we add a separate test for the multi-setting?

Reply:
This should be exactly this test, if I'm understanding you correctly.

Here we're setting two classes at once on an element that doesn't have any CSS classes set prior to this (see previous assertion).

Reply:
Yeah, I see that we are testing the setting of multiple values here, but the emphasis should have been on **separate** in the previous message. 😉